### PR TITLE
Python 2 compatibility review

### PR DESCRIPTION
**Pull request by deniskristak**
https://github.com/easybuilders/easybuild-docs/issues/169

added note that support for python 3.5 and 2.* will be dropped in easybuild 5.0

### CODE REVIEW

File: docs/python-2-3-compatibility.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ compatible with both Python versions 2 and 3. More specifically, the
 following Python versions are currently supported:
 
 - Python 2.6.x (support was removed in EasyBuild v4.4.0)
-- Python 2.7.x
-- Python 3.5.x
+- Python 2.7.x (support will be removed in EasyBuild v5.0.0)
+- Python 3.5.x (support will be removed in EasyBuild v5.0.0)
```
Comment (boegel) [inline docs/python-2-3-compatibility.md:9]: link to `easybuild-v5/overview-of-changes.md`

**Review by boegel — CHANGES_REQUESTED** [review]

File: docs/python-2-3-compatibility.md (hunk 1 above)
Comment (deniskristak) [reply docs/python-2-3-compatibility.md:9]: that file has `# Overview of changes in EasyBuild version 5.0 {: #overview }`  line in it, where the `overview` tag is quite not specific...should I change it to sth like `eb5_overview` ?

Comment (boegel) [reply docs/python-2-3-compatibility.md:9]: Either rename to `eb5_changes_overview` (like in `easybuild-v4/overview-of-changes.md`), or just drop it, a tag on top of the page isn't very useful anyway

Comment (boegel) [reply docs/python-2-3-compatibility.md:9]: @deniskristak ping?

**Review by boegel — CHANGES_REQUESTED** [review]

File: docs/python-2-3-compatibility.md [hunk 2]
```diff
@@ -5,8 +5,9 @@ compatible with both Python versions 2 and 3. More specifically, the
 following Python versions are currently supported:
 
 - Python 2.6.x (support was removed in EasyBuild v4.4.0)
-- Python 2.7.x
-- Python 3.5.x
+- Python 2.7.x (support will be removed in EasyBuild v5.0.0)
```
Comment (boegel) [inline docs/python-2-3-compatibility.md:8]:
```suggestion
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Pull request merged**